spec/unit/lib/bosh_errand_environment_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+RSpec.describe BoshErrandEnvironment do
+  let(:stdout_sink_enabled) { true }
+  let(:config) { VCAP::CloudController::Config.new({ db: DbConfig.new.config, logging: { level: 'debug2', stdout_sink_enabled: stdout_sink_enabled } },
```

Comment:
Is `db: DbConfig.new.config` needed? The `load_models` function is mocked anyway...

Reply:
The formatting looks weird. If the `db` config is not needed, it might fit into a single line again...

Reply:
`setup_environment` runs `@config.get(:db)`, so we need it.